### PR TITLE
defining offsets because its shit

### PR DESCRIPTION
**Pull request by ScureX**
hello smart fellas and fart smellas, basically talked abt this with gecko. we keep using offsets and its fucking awful how we just put a buncha random ass hex numbers in there. made a header file with the 2 offsets i know, this is just a start so people who know way more than i do can put in their offsets and replace all shitty hex numbers

---

image

### CODE REVIEW

File: NorthstarDedicatedTest/serverauthentication.cpp [hunk 1]
```diff
@@ -212,7 +213,7 @@ bool ServerAuthenticationManager::AuthenticatePlayer(void* player, int64_t uid,
 		{
 			authFail = false;
 			// uuid
-			strcpy((char*)player + 0xF500, strUid.c_str());
+			strcpy((char*)player + 0xF00, strUid.c_str());
```
Comment (GeckoEidechse) [inline NorthstarDedicatedTest/serverauthentication.cpp:216]:
```suggestion
			strcpy((char*)player + OFFSET_PLAYER_UID, strUid.c_str());
```
Missed/typoed one ;)

Comment (ScureX) [reply NorthstarDedicatedTest/serverauthentication.cpp]: forgor 💀

**Comment by BobTheBob9** [comment]
we do this with proper struct defs on refactor already

**Comment by ScureX** [comment]
> we do this with proper struct defs on refactor already

in how many years will that be finally merged ^^'

**Comment by BobTheBob9** [comment]
refactor is merged

**Pull request closed without merging**